test/unit/rubygem_test.rb

```diff
@@ -3,7 +3,7 @@
 class RubygemTest < ActiveSupport::TestCase
   context "with a saved rubygem" do
     setup do
-      @rubygem = create(:rubygem, name: "SomeGem")
+      @rubygem = Rubygem.new(name: "SomeGem")
```

Comment:
:thinking: Any reason to not use `build` here?

Reply:
I can if you want. I just prefer not using factories for building where .new works just as well. But happy to respect project style if that’s an issue.

Reply:
I see `build` used across tests and I'm just curious if there's any particular reason to not use it (like performance wise). I have no strong opinion on this.